router/throttler/adaptive.go

```diff
@@ -72,6 +74,10 @@ func (t *adaptiveThrottler) getLimit() int64 {
 	if t.limitFactorMeasurement != nil {
 		t.limitFactorMeasurement.Gauge(limitFactor)
 	}
-	limit := int64(float64(t.config.maxLimit.Load()) * limitFactor)
+	maxLimit := t.config.maxLimit.Load()
+	if maxLimit == 0 {
+		maxLimit = t.config.staticLimit.Load() * 130 / 100 // increase static limit by 30% if maxLimit is not set
```

Comment:
Can we also have this multiplier configurable & hot reloadable? We could also start with `150%`

Reply:
Done